### PR TITLE
BetaBinomial variance is not correctly described

### LINKED ISSUE #6495: DOC: BetaBinomial variance is not correctly described
### Issue with current documentation:

In [pymc.BetaBinomial docstring](https://www.pymc.io/projects/docs/en/stable/_modules/pymc/distributions/discrete.html#BetaBinomial) the variance is described as follows:

Variance  :math:`n \dfrac{\alpha \beta}{(\alpha+\beta)^2 (\alpha+\beta+1)}`

But, as indicated in the [Wikipedia page](https://en.wikipedia.org/wiki/Beta-binomial_distribution) for the beta-binomial distribution, it should be:

Variance  :math:`\dfrac{n \alpha \beta (\alpha+\beta+n)}{(\alpha+\beta)^2 (\alpha+\beta+1)}`

### Idea or request for content:

_No response_

### PR DESCRIPTION
**Pull request by soma2000-lang**
Fixes #6495
@OriolAbril @ricardoV94

### CODE REVIEW

File: pymc/distributions/discrete.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ def BetaBinom(a, b, n, x):
     ========  =================================================================
     Support   :math:`x \in \{0, 1, \ldots, n\}`
     Mean      :math:`n \dfrac{\alpha}{\alpha + \beta}`
-    Variance  :math:`n \dfrac{\alpha \beta}{(\alpha+\beta)^2 (\alpha+\beta+1)}`
+    Variance  :math:`\dfrac{n \alpha \beta (\alpha+\beta+n)}{(\alpha+\beta)^2 (\alpha+\beta+1)}
```
Comment (OriolAbril) [inline pymc/distributions/discrete.py:223]: The formula needs a closing `` ` `` in order to be rendered properly. Other than that it looks good, thanks!

Here is the preview for this PR: https://pymcio--6498.org.readthedocs.build/projects/docs/en/6498/api/distributions/generated/pymc.BetaBinomial.html

**Review by OriolAbril — CHANGES_REQUESTED** [review]

**Comment by soma2000-lang** [comment]
@OriolAbril It would be great if you can force push the change required ,beacause I somehow messed up with git and had to delete my forked repository ,even the manual edit option is not available.Really Sorry.

Regards

**Comment by larryshamalama** [comment]
> @OriolAbril It would be great if you can force push the change required ,beacause I somehow messed up with git and had to delete my forked repository ,even the manual edit option is not available.Really Sorry.
> 
> Regards

Hi @soma2000-lang,  I'd thought that I'd provide some steps in recovering your current work in progress. I've also previously ran into the issue that you're currently experiencing.

#### 1. Add upstream

With this pull request created, you should be able to fetch it from the `pymc-devs` PyMC repo as an upstream. Try the following:

```bash
git remote -v # checks your origin and upstream repositories
git remote add upstream https://github.com/pymc-devs/pymc/
```

You may already have the upstream set, so this step may be optional.

#### 2. Fetch pull request

With the upstream set, run the following:

```
git fetch upstream pull/6498/head:patch-1
```

#### 3. Checkout

```
git checkout patch-1
```

After the checkout, you should find yourself back on the branch here and can now continue making edits.

Let us know if you have any questions!

**Comment by soma2000-lang** [comment]
Thanks @larryshamalama I will try

**Pull request closed without merging**